### PR TITLE
alpha fixes

### PR DESCRIPTION
**Pull request by realcodywburns**
issue #42 added showAccessButton: true to onDrop req password
issue #46 added showInfo on wallet.js

### CODE REVIEW

File: src/components/wallet/open.js [hunk 1]
```diff
@@ -94,52 +97,62 @@ class WalletForm extends React.Component {
...
                 <Radio name="keyFormat" value="text" onChange={this.handleFormat}>
                   Plain Text Private Key</Radio>
               </FormGroup>
             </Form>
           </Col>
-          {this.state.showTextKey && <Col sm={12} md={4}>
+          {this.state.showIntro && <Col sm={12} md={4} lg={4}>
+          <Well>
+          <p>
+          Please provide the private key for the address that you would like to use to create your ERC 223 tokens. The address you choose requires a minimum amount of ETC or ETH to pay for the Gas (Transaction Fee) necessary to execute the smart contract which creates the tokens you wish to create. We recommend having at least .1 ETC or .1 ETH in order to execute the smart contract and be able to move your tokens when you wish.
```
Comment (elaineo) [inline src/components/wallet/open.js]: let's leave any mention of ETH out until we actually support it (don't want ppl losing ETH trying to create tokens here)

File: package.json [hunk 2]
```diff
@@ -12,6 +12,7 @@
     "bignumber.js": "^4.0.2",
     "ethereumjs-tx": "^1.3.1",
     "ethereumjs-util": "^5.1.2",
+    "ethjs-util": "^0.1.4",
```
Comment (elaineo) [inline package.json]: Is this actually used anywhere?

Comment (realcodywburns) [inline package.json]: When unlocking a wallet i get errors if it is not there

Comment (elaineo) [inline package.json]: but it's not actually being imported anywhere

Comment (elaineo) [inline package.json]: oh, i see the problem... there are two versions of ethUtil being used by the wallet library. my fault, i will fix.

**Pull request merged**